gap/SkeletalFinSetsForCAP.gi

```diff
@@ -49,7 +49,7 @@ InstallMethodForCompilerForCAP( FinSetOp,
   function ( cat, n )
     local int;
     
-    int := ObjectifyObjectForCAPWithAttributes( rec( ), cat, Length, n );
+    int := ObjectifyObjectForCAPWithAttributes( rec( ), cat, Length, n, AsList, [ 1 .. n ] );
```

Comment:
This is suboptimal for the compiler for two reasons:

1. The compilation time doubles (at least for operations returning objects) because `n` is compiled twice and only deduped at the very end.
2. The compiler cannot (yet?) pull `ObjectifyObjectForCAPWithAttributes` with more than one attribute out of iterations (e.g. `CapJitFixpoint`).

However, if you think the advantages outweigh the disadvantages, we can keep it.

Reply:
I replaced all AsList occurrences for objects with `[ 1 .. Length( object ) ]`.